src/vs/editor/common/view/editorColorRegistry.ts

```diff
@@ -19,6 +19,7 @@ export const editorCursorBackground = registerColor('editorCursor.background', n
 export const editorWhitespaces = registerColor('editorWhitespace.foreground', { dark: '#e3e4e229', light: '#33333333', hc: '#e3e4e229' }, nls.localize('editorWhitespaces', 'Color of whitespace characters in the editor.'));
 export const editorIndentGuides = registerColor('editorIndentGuide.background', { dark: editorWhitespaces, light: editorWhitespaces, hc: editorWhitespaces }, nls.localize('editorIndentGuides', 'Color of the editor indentation guides.'));
 export const editorLineNumbers = registerColor('editorLineNumber.foreground', { dark: '#5A5A5A', light: '#2B91AF', hc: Color.white }, nls.localize('editorLineNumbers', 'Color of editor line numbers.'));
+export const editorActiveLineNumber = registerColor('editorActiveLineNumber.foreground', { dark: null, light: null, hc: null }, nls.localize('editorActiveLineNumber', 'Color of editor active line number'));
```

Comment:
I'm thinking if we should have this feature by default, by assigining proper `foreground` for default themes.

Reply:
Are you suggesting to assign colors for shipped with vscode themes: [monokai, red, ...] or default values for all themes? Assigning a default for all themes will look weird for a lot of themes: For instance, if I assign some gray color for all dark themes as default and a theme has a blue(or some other distant from gray) it would really stand out.

Reply:
I agree with @usernamehw to use 'null' as default here. That way we don't break anyone.

Reply:
I'm convinced that null is a good value.